### PR TITLE
feat: added the acs config

### PR DESCRIPTION
**Pull request by sachaudh**
This PR is to add the Advanced Cluster Security (ACS) app to the consoledot environment

### CODE REVIEW

File: main.yml [hunk 1]
```diff
@@ -11,6 +11,18 @@ access-requests:
     paths:
       - /internal/access-requests
 
+acs:
+  title: Advanced Cluster Security
+  api:
+    version:
```
Comment (karelhala) [inline main.yml]: A typo
```suggestion
    versions:
```

File: main.yml [hunk 2]
```diff
@@ -11,6 +11,18 @@ access-requests:
...
+      - v1
+  description: Frontend for Red Hat employees to manage ACS instances
+  deployment_repo: https://github.com/stackrox/acs-ui-build
```
Comment (karelhala) [inline main.yml:20]: @ryelo Are there any drwbacks that the build repo is outside our org? AFAIK it shouldn't be a problem, just maintenance reasons.

File: main.yml [hunk 3]
```diff
@@ -11,6 +11,18 @@ access-requests:
...
+  frontend:
+    paths:
+      - /acs
```
Comment (karelhala) [inline main.yml]: This should include the full path
```suggestion
      - /application-services/acs
```

File: chrome/fed-modules.json [hunk 4]
```diff
@@ -623,15 +623,27 @@
...
+    "acs": {
+        "manifestLocation": "/apps/acs/fed-mods.json",
+        "modules": [
           {
-              "id": "sandbox",
+              "id": "acs",
               "module": "./RootApp",
               "routes": [
-                  "/openshift/sandbox"
+                  "/acs"
```
Comment (karelhala) [inline chrome/fed-modules.json]: This should include the full path
```suggestion
                  "/application-services/acs"
```

**Review by karelhala — COMMENTED** [review]
Looking good! Just a few routing adjustments and one typo.

File: main.yml (hunk 2 above)
Comment (Hyperkid123) [reply main.yml:20]: I am helping @sachaudh with the build repo. We will see if Jenkins picks up the changes. If not we will discuss an acceptable solution.

Comment (sachaudh) [reply main.yml:20]: If it's better to keep it within the org, would there be any trouble in moving the repo after adding this? If so, I can try doing that before

Comment (karelhala) [reply main.yml:20]: I am just worried for jenkins to pick up the changes. But if jenkins will be fine we def can keep it where it is right now (no need to stall this any further)

**Comment by karelhala** [comment]
I don't have any objections to the code or anything, @sachaudh do you want us to merge this so you can test if everything is running?

**Comment by sachaudh** [comment]
> I don't have any objections to the code or anything, @sachaudh do you want us to merge this so you can test if everything is running?

Yes, that would be very much appreciated 🙏🏼

File: main.yml (hunk 2 above)
Comment (ryelo) [reply main.yml:20]: AFAIK the build repo has to be within the RedHatInsights github org. There shouldn't be any trouble moving the repo if it doesn't pick it up.

**Comment by Hyperkid123** [comment]
@sachaudh lets see if the Jenkins pipeline picks up the changes

File: main.yml (hunk 2 above)
Comment (sachaudh) [reply main.yml:20]: No worries. I'm not an organization member so what should I do to create a repo under RedHatInsights?

**Comment by sachaudh** [comment]
> @sachaudh lets see if the Jenkins pipeline picks up the changes

@Hyperkid123 how can I test to see if it worked. I wasn't able to load It locally before because I didn't configure it in the cloud-services-config. Now that it's in, I noticed that the side nav bar still didn't have the link and it redirects me saying that path doesn't exist

Screen Shot 2022-03-14 at 10 04 59 AM

**Comment by sachaudh** [comment]
Oops my bad, so it does work actually. I was supposed to check `ci.cloud.redhat.com/beta`. Nice 👍🏼 

Screen Shot 2022-03-14 at 10 10 27 AM

But yea, I'm wondering how can I get it working locally with the new appUrl?

**Comment by Hyperkid123** [comment]
@sachaudh 

> Oops my bad, so it does work actually. I was supposed to check ci.cloud.redhat.com/beta. Nice 👍🏼

You should not use ci/qa environments. If you can use `console.stage.redhat.com/beta`. It shares the config with ci and qa

> But yea, I'm wondering how can I get it working locally with the new appUrl?

That depends on your local configuration.
https://github.com/stackrox/acs-ui/blob/main/config/dev.webpack.config.js The config looks OK. I think you just have to run the app in the BETA env.  You want to start the dev server with the `npm run:start:beta` command.

**Pull request merged**